components/Modal/CreateRecipe.js

```diff
@@ -40,16 +42,22 @@ class CreateRecipe extends React.Component {
     const o = Object.assign({}, this.state.recipe);
     o[prop] = e.target.value;
     this.setState({ recipe: o });
+    if (prop === 'name') {
+      this.dismissErrors();
+      this.handleErrors(e.target.value);
```

Comment:
I have added this recommendation to the most recent commit. It appears to fix the issue.